pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -356,6 +365,26 @@ func buildConfigApplyJob(comp *vshnv1.VSHNKeycloak, adminSecret, jobName string)
 					RestartPolicy:    corev1.RestartPolicyOnFailure,
 					ImagePullSecrets: []corev1.LocalObjectReference{{Name: pullsecretName}},
 					Volumes:          volumes,
+					// We need this init container as the apply-config container could otherwise start a parallel keycloak setup
```

Comment:
Is there a reason to create this job during provisioning? As I understood this initContainer exists only to make sure during provisioning there are no parallel keycloak setup, right?

Reply:
That works for me.